.github/workflows/tests-smoke.yaml

```diff
@@ -187,8 +188,11 @@ jobs:
           cilium_pod=$(kubectl -n kube-system get po -o name --field-selector=status.phase==Running -l 'k8s-app=cilium' -o jsonpath='{.items[0].metadata.name}' )
           kubectl -n kube-system exec $cilium_pod -- sh -c "apt update && apt install curl -y"
           kubectl -n kube-system exec $cilium_pod -- curl http://localhost:9090/metrics > metrics.prom
-          go install github.com/prometheus/prometheus/cmd/promtool@v2.31.1
-          cat metrics.prom | $HOME/go/bin/promtool check metrics
+          curl -sSLO https://github.com/prometheus/prometheus/releases/download/v${PROM_VERSION}/prometheus-${PROM_VERSION}.linux-amd64.tar.gz
```

Comment:
Should there be a comment that points to the Prometheus issue, so that we can restore simpler steps once it's addressed?